docs/views/about.html

```diff
@@ -24,15 +24,45 @@ <h1 class="govuk-heading-xl">About</h1>
       <p class="govuk-body-l">
         The GOV.UK Prototype Kit provides a simple way to make interactive prototypes that look like pages on GOV.UK. These prototypes can be used to show ideas to people you work with, and to do user research
       </p>
-
-      <h2 class="govuk-heading-m">Community</h2>
-
-      <p>We have two Slack channels for the GOV.UK Prototype Kit. You'll need a government email address to join.</p>
+      
+      <h2 class="govuk-heading-m">Support</h2>
+      
+      <p>The GOV.UK Prototype Kit is maintained by the Government Digital Service. If you’ve got a question or need support you can:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li><a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit/">Slack channel for users of the GOV.UK Prototype Kit</a></li>
-        <li><a href="https://ukgovernmentdigital.slack.com/messages/prototype-kit-dev/">Slack channel for developers of the GOV.UK Prototype Kit</a></li>
+        <li>email 
+          <a href="mailto:govuk-design-system-support@digital.cabinet-office.gov.uk" class="govuk-link" data-hsupport="email">
```

Comment:
The data attributes are for analytics tracking on the Design System site – I don't know if we're planning on tracking the links on the Prototype Kit site. Can we check with Daisy and remove these if they're not required?

Reply:
I decided not to ask Daisy as we aren't tracking anything. I assume that this would be looked at later when its needed.

Reply:
In which case can we remove these data attributes? Ta!